applications/StructuralMechanicsApplication/StructuralMechanicsApplication.py

```diff
@@ -4,7 +4,7 @@
 application_folder = "StructuralMechanicsApplication"
 
 # The following lines are common for all applications
-from . import application_importer
+from .. import application_importer
```

Comment:
I am not sure if this also works in python2
=> if not I think it can be fixed with little effort